examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -135,7 +135,8 @@ class TappableTravelDestinationItem extends StatelessWidget {
                   onTap: () {
                     print('Card was tapped');
                   },
-                  splashColor: Theme.of(context).colorScheme.primary.withAlpha(30),
+                  splashColor: Theme.of(context).colorScheme.primary.withOpacity(0.12),
```

Comment:
Add comment explaining .12 if you can. Maybe link to guidelines.

Reply:
done